docs/Options.md

```diff
@@ -207,11 +207,24 @@ Go section is shown only in directories that contain `Godeps`, `glide.yaml`, any
 | Variable | Default | Meaning |
 | :------- | :-----: | ------- |
 | `SPACEFISH_GOLANG_SHOW` | `true` | Show current Go version or not |
-| `SPACEFISH_GOLANG_PREFIX` | `$SPACEFISH_PROMPT_DEFAULT_PREFIX` | Prefix before the Go section |
+| `SPACEFISH_GOLANG_PREFIX` | ` ` | Prefix before the Go section |
 | `SPACEFISH_GOLANG_SUFFIX` | `$SPACEFISH_PROMPT_DEFAULT_SUFFIX` | Suffix after the Go section |
 | `SPACEFISH_GOLANG_SYMBOL` | `🐹·` | Character to be shown before Go version |
 | `SPACEFISH_GOLANG_COLOR` | `cyan` | Color of Go section |
 
+
+### Kubectl \(`kubectl`\)
```

Comment:
Like in the docs of spaceship, I would change the text here to `Kubectl context (kubecontext)` 😄